### PR TITLE
[RC2] Update Kubernetes and Functest releases in CNTT RC2

### PR DESCRIPTION
**Pull request by collivier**
It also updates all test result urls and 2 outdated tags in
doc/ref_cert/README.md.

The test case list will be updated in a next PR.

Signed-off-by: Cédric Ollivier <cedric.ollivier@orange.com>

### CODE REVIEW

**Review by gkunz — APPROVED** [review]

**Review by michaelspedersen — APPROVED** [review]

File: doc/ref_cert/README.md [hunk 1]
```diff
@@ -144,10 +144,10 @@ CNTT requirements about verification, validation, compliance, and conformance:
   for third-party conformance review
 
 Here are a couple of publicly available playbooks :
-- [Xtesting samples](https://git.opnfv.org/functest-xtesting/plain/ansible/site.yml?h=stable/leguer)
-- [OpenStack verification](https://git.opnfv.org/functest/plain/ansible/site.yml?h=stable/leguer)
-- [CNTT RC1](https://git.opnfv.org/functest/plain/ansible/site.cntt.yml?h=stable/hunter)
-- [Kubernetes verification](https://git.opnfv.org/functest-kubernetes/plain/ansible/site.yml?h=stable/leguer)
+- [Xtesting samples](https://git.opnfv.org/functest-xtesting/plain/ansible/site.yml?h=stable/wallaby)
+- [OpenStack verification](https://git.opnfv.org/functest/plain/ansible/site.yml?h=stable/wallaby)
```
Comment (pgoyal01) [inline doc/ref_cert/README.md:148]: @collivier Since this is for Kali release, should the OpenStack release be "wallaby"?

Comment (collivier) [reply doc/ref_cert/README.md:148]: I considered both options.
Here it's about any OpenStack verification and then I selected the latest Functest stable release.
In any case, I would consider Xtesting wallaby is the latest answer here.

@pgoyal01 
I can switch to Jerma if it helps the reader. not a big deal. Waiting your answer.

**Review by rihabbanday — APPROVED** [review]

File: doc/ref_cert/README.md (hunk 1 above)
Comment (pgoyal01) [reply doc/ref_cert/README.md:148]: @collivier Thanks -- let us leave it as Wallaby.

**Review by pgoyal01 — APPROVED** [review]

**Comment by pgoyal01** [comment]
@walterkozlowski Ready to merge

**Pull request merged**